src/spec/lib/TwaManifestSpec.ts

```diff
@@ -45,9 +45,9 @@ describe('TwaManifest', () => {
       expect(twaManifest.iconUrl)
           .toBe('https://pwa-directory.com/favicons/android-chrome-512x512.png');
       expect(twaManifest.maskableIconUrl).toBeUndefined();
-      expect(twaManifest.themeColor).toBe('#7cc0ff');
-      expect(twaManifest.navigationColor).toBe('#7cc0ff');
-      expect(twaManifest.backgroundColor).toBe('#7cc0ff');
+      expect(twaManifest.themeColor.hex()).toBe('#7CC0FF');
```

Comment:
Would it be worth having each of the colours be different, so we can catch if we accidentally mix them up somewhere?